apps/cms/src/app/pages/welcome/welcome.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'th-welcome',
+  templateUrl: './welcome.component.html',
+  styleUrl: './welcome.component.less',
```

Comment:
🔍 I don't find it useful to generate empty files and linking them to `styleUrl` since that property is optional. Is there a convention around this?

Reply:
Sure. But we'll delete this whole component soon anyway.